### ISSUE TITLE
Problems Panel: Show item value in List layout mode

### ISSUE THREAD

**Issue by Yhache** [grafana/grafana-zabbix#721]
In Zabbix, there is an option that you can check that allows you to see what is the latest value retrieved from the agent for any problems. Example: Low disk space on C:\ , latest value : 3.65Gb.

In the settings of the Problem panel, in the "Show Fields" section, have the option of getting that last value received from the agent regarding a problem"

**Comment by alexanderzobnin** [comment]
You can see the item value in the dropdown area:

![Screenshot from 2019-04-18 18-49-41](https://user-images.githubusercontent.com/4932851/56373965-cf61d100-620a-11e9-8852-48665f7508d8.png)

**Comment by Yhache** [comment]
Yes thats right, but this can only be accessed by the Table layout... i'd love to this as another column if it's possible. I don't see that info on the List layout either
